source/_integrations/proxmoxve.markdown

```diff
@@ -22,6 +22,8 @@ After configuring this integration, the binary sensors automatically appear.
 
 <div class='note'>
 You should have at least one VM or container entry configured, else this integration won't do anything.
+  These instructions are deprecated and better suited to follow this link to get up and running.
```

Comment:
We should not send user to third-party resources to set up an core integration.

If our existing instructions needs to be update, please update this page instead.

Reply:
Just one note: I am not aware this integration has been deprecated. Could you point me to a resource that made this decision?